inst/metadata/advanced_run_options.json

```diff
@@ -214,6 +214,36 @@
 					"helpText": "t_(OPTIONS_ADVANCED_LOGIT_HELP)"
 				}
 			]
+		},
+	        {
+		       "label": "t_(OPTIONS_ADVANCED_APPROX_DEFF_LABEL)",
+                       "helpText": "t_(OPTIONS_ADVANCED_APPROX_DEFF_LABEL)",
```

Comment:
Sorry another thing came up after putting this through our schema. You can't have help text here. Help text goes on the individual controls only.

Reply:
Thanks -- removed in 7144474. (Also that is a pretty nice commit hash.)